tsconfig.json

```diff
@@ -24,5 +24,15 @@
   "ts-node": {
     "swc": true,
   },
-  "exclude": ["dist", "node_modules", "out", "**/*.mts"],
+  "include": [
```

Comment:
I believe this was intentionally removed

Reply:
can you explain. @dmaretskyi ?

Reply:
This was changed 1-2 weeks ago, the idea being that the tsconfig files from individual projects should be including sources files, while the root one should include nothing. In thereory that should have improved the performance of tsserver, but personally it's still quite unstable for me